### PR TITLE
Handle overflow of content in PreviewWidget

### PR DESCRIPTION
**Pull request by AndriiTsarenko**
### 🔦 Summary
Adding a prop for a scrollable content within `<PreviewWidget/>` 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5897/

File: src/PreviewWidget/PreviewWidget.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { skins, dataHooks } from './constants';
 import { stVars as colors } from '../Foundation/stylable/colors.st.css';
+import classNames from 'classnames';
```
Comment (TalZa1810) [inline src/PreviewWidget/PreviewWidget.js:5]: No need to use classNames when using stylable

File: src/PreviewWidget/PreviewWidget.js [hunk 2]
```diff
@@ -70,7 +76,12 @@ class PreviewWidget extends React.PureComponent {
         data-hook={dataHook}
         style={rootStyles}
       >
-        <div data-hook={dataHooks.contentArea} className={classes.contentArea}>
+        <div
+          data-hook={dataHooks.contentArea}
+          className={classNames(classes.contentArea, {
+            [classes.contentAreaScrollable]: scrollable,
```
Comment (TalZa1810) [inline src/PreviewWidget/PreviewWidget.js:82]: This is basically a state which can be passed within the root div. You can check out the `contentOutline`  stylable state as a reference.

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Hey, just some few comments

File: src/PreviewWidget/index.d.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ export interface PreviewWidgetProps {
   backgroundColor?: string;
   height?: string;
   width?: string;
+  scrollable?: boolean;
```
Comment (EgoziE) [inline src/PreviewWidget/index.d.ts:14]: Please add to type test file PreviewWidget.tsx

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by milkyfruit — APPROVED** [review]
Looks good from design side.

**Pull request merged**